src/react/databrowser/objects/details/Metadata.jsx

```diff
@@ -126,25 +140,25 @@ function Properties({ objectMetadata }: Props) {
                 <Items>
                     {
                         items.map((p, i) => {
-                            const isShrink = isMetaWord(p.key);
-                            return <Item isShrink={isShrink} key={i}>
+                            const isUserMD = isUserType(p.type);
+                            return <Item isShrink={isUserMD} key={i}>
```

Comment:
is using the index for a react key ideal? wouldn't you want to use something that is unique despite order? Not sure on this but I remember always using some sort of value when I was using react.

Reply:
We don’t have "stable IDs" for those rendered items (something like p.id). 
Also, in this case, the order of items should not change.